README.md

```diff
@@ -27,6 +31,54 @@ It can be used in one of the following ways.
 - Use `-debug` of `--debug` option
 - Set `SD_CMD_DEBUG_LOG` environment variable to `true`
 
+### Publish
+We publish the command specified by a yaml.
+```bash
+USAGE:
+   sd-cmd publish [command options]
+
+OPTIONS:
+   -f, --f    Specify the path of yaml to publish (default: sd-command.yaml)
+   -t, --t    Specify the tag given to the command (default: sd-command.yaml)
+
+EXAMPLE:
+   sd-cmd publish -f ./sd-command.yaml -t latest
+```
+
+### Validate
+We validate that a yaml is in the correct format for the command.
+```bash
+USAGE:
+   sd-cmd validate [command options]
+
+OPTIONS:
+   -f, --f    Specify the path of yaml to validate (default: sd-command.yaml)
+
+EXAMPLE:
+   sd-cmd validate -f ./sd-command.yaml
+```
+
+### Promote
+We give the tag specified by `tag` to the version specified by `targetVersion`. The tag will be removed from the version which `tag` is originally assigned to. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).
```

Comment:
```suggestion
We give the tag specified by `tag` to the version specified by `targetVersion`. The tag will be removed from the version which `tag` is originally assigned to. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).

Giving a `tag` for a `targetVersion` of command. If a `tag` is already set to another version, that tag will be moved to `targetVersion`. `targetVersion` can be set exact version or tag (e.g. 1.0.1, latest).
```

Reply:
The following is right?
---
Giving a `tag` to a `targetVersion` of command.
---

Reply:
It may be better.